README.md

```diff
@@ -34,17 +34,19 @@ $ pip install -e .
 | `TYP003` | Missing type annotation for `**kwargs`        |
 
 ### Method Annotations
-| ID       | Description                                        |
-|----------|----------------------------------------------------|
-| `TYP101` | Missing type annotation for `self` in class method |
-| `TYP102` | Missing type annotation for `cls` in classmethod   |
+| ID       | Description                                           |
+|----------|-------------------------------------------------------|
+| `TYP101` | Missing type annotation for `self` in class method    |
```

Comment:
Calling it a `class method` is confusing cause it's too similar to `TYP103`. Yes, from context it could be figured out. However, I think calling it `method` should be sufficient as that already implies it's part of a class (opposed to a function).

Reply:
```suggestion
| `TYP101` | Missing type annotation for `self` in method    |
```

Agreed. Let's just call it a method.

Reply:
@sco1 You forgot to apply this change to the README